downstream/modules/platform/con-about-automation-mesh.adoc

```diff
@@ -1,10 +1,10 @@
 
 [id="con-automation-mesh"]
 
-= About automation mesh
+= About {MeshName}
 
 [role="_abstract"]
-Automation mesh is an overlay network intended to ease the distribution of work across a large and dispersed collection of workers through nodes that establish peer-to-peer connections with each other using existing networks.
+{AutomationMeshStart} is an overlay network intended to ease the distribution of work across a large and dispersed collection of workers through nodes that establish peer-to-peer connections with each other using existing networks.
 
 {PlatformName} 2 replaces Ansible Tower and isolated nodes with {ControllerName} and {HubName}. {ControllerNameStart} provides the control plane for automation through its UI, Restful API, RBAC, workflows and CI/CD integration, while Automation Mesh can be used for setting up, discovering, changing or modifying the nodes that form the control and execution layers.
```

Comment:
There's an Automation Mesh here. Attribute?

Reply:
Ah, I it didn't get captured as it's Automation Mesh as opposed to automation mesh. I'll double check the code with that in mind.